__tests__/integration/hathorwallet_facade.test.js

```diff
@@ -1135,6 +1461,13 @@ describe('mintTokens', () => {
       100,
     );
 
+    // Should not mint more tokens than the HTR funds allow
```

Comment:
Nice one, we should do the same for melt. Try to melt more tokens than you have.

Reply:
Added this test on [0916488](https://github.com/HathorNetwork/hathor-wallet-lib/pull/409/commits/0916488c411ea9ebe775714fd2fd57428a925e4a).